package.json

```diff
@@ -5,7 +5,8 @@
   "main": "dist/itowns2.js",
   "scripts": {
     "lint": "eslint \"src/**/*.js\" \"test/**/*.js\"",
-    "test": "npm run lint && npm run build",
+    "doc": "jsdoc src/Core/Commander/Interfaces/ApiInterface/* -d doc/API_Doc",
+    "test": "npm run lint && npm run build && npm run doc",
```

Comment:
Don't generate docs in `npm test`.

Actually, I'd rather generate the docs during the `after_success` too, so possibly call it from the `autodeploy.sh` instead of the current copy? or from the `.travis.yml` before running `autodeploy.sh` (maybe change the `npm run doc` to *not* pass the `-d` and then invoke it as `npm run doc -- -d out/doc/API_Doc`?)